docs/en_us/ingress/rewrite.md

```diff
@@ -0,0 +1,247 @@
+# Rewrite
+
+The BFE engine provides multiple ways to modify the url of HTTP request, including modifying host, path and query.
+
+The BFE Ingress Controller supports parsing the related annotations and modifying the url of matched requests. 
+
+## How to Config
+
+* In the Ingress object,
+  - `spec.rules` define the route rules
+  - `metadata.annotations` define the rewrite actions for modifying the matched requests
+
+Reference format:
+
+```yaml
+metadata:
+  annotations:
+    bfe.ingress.kubernetes.io/rewrite-url.host: '[{"params": "baidu.com", "when": "AfterLocation"}]'
+    bfe.ingress.kubernetes.io/rewrite-url.path: '[{"params": "/bar"}]'
+    bfe.ingress.kubernetes.io/rewrite-url.query-rename: >-
+     [
+       {
+          "params": {"name": "user"}, 
+          "when": "AfterLocation", 
+          "order": 1
+       }
+     ]
+spec:
+  rules:
+  - ...
+```
+
+### Description of Rewrite Rule
+
+| Config Item           | Description                                                  |
+| --------------------- | ------------------------------------------------------------ |
+| callbackList[]        | A list of rewrite rule, each rule must have an unique callback point. |
+| callbackList[].params | Params of rewrite rule.                                      |
+| callbackList[].order  | The order of rewrite rule,  effective on the same callback point. |
+| callbackList[].when   | The callback point of rewrite url, default is `AfterLocation` (only support this callback point) |
+
+## Rewrite Host
+
+The BFE Ingress Controller supports for configuring host information by annotations, including set static host or dynamic host.
```

Comment:
The BFE Ingress Controller supports configuring rewrite of "host" by annotations, including setting static host or dynamic host.